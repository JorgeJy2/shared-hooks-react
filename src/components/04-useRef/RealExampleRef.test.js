import {
    shallow
} from "enzyme";
import {
    RealExampleRef
} from "./RealExampleRef";

describe('real example ref', () => {
    const wrapper = shallow( < RealExampleRef / > );
    test('should shapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });

    test('should component default MultipleCustomHooks', () => {
        expect(wrapper.find('MultipleCustomHooks').exists()).not.toBeTruthy();
    });

    test('should component MultipleCustomHooks', () => {
        const button = wrapper.find('button');
        button.simulate('click');
        expect(wrapper.find('MultipleCustomHooks').exists()).toBeTruthy();
    });
});