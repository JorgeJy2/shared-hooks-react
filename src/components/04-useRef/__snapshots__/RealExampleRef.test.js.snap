// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`real example ref should shapshot 1`] = `
<Fragment>
  <h1>
    RealExampleRef
  </h1>
  <hr />
  <br />
  <button
    className="btn btn-success m-2"
    onClick={[Function]}
  >
    Show / Hide
  </button>
</Fragment>
`;
